package.json

```diff
@@ -28,7 +28,7 @@
     "babel-eslint": "6.1.2",
     "babel-plugin-add-module-exports": "0.2.1",
     "eslint": "3.4.0",
-    "eslint-plugin-mocha": "4.5.1",
+    "eslint-plugin-mocha": "4.4.0",
```

Comment:
Would you mind elaborating on why this change is required? Is this what caused us to add the `package-lock.json` above? I'm inclined to think this should be pulled out into a separate PR, but I might be missing some of the context.

Reply:
Ah yeah I wanted to pull it out into another PR as well, but it was required to make CI pass. But maybe I can pull this into another PR first.